packages/shared/src/utils/Tabs.tsx

```diff
@@ -28,6 +28,7 @@ type TabsProps = {
   id: string;
   basePath?: string; // The page that is common to these tabs
   updatePathOnSelect?: boolean;
+  customData?: unknown;
```

Comment:
At least we'll know we have to provide an object...
```suggestion
  customData?: object;
```

Reply:
it could be array as well... anything is possible... `{}` is just default...

Reply:
Ack.